eng/versioning/version_client.txt

```diff
@@ -343,6 +343,7 @@ com.azure.resourcemanager:azure-resourcemanager-devspaces;1.0.0-beta.2;1.0.0-bet
 com.azure.resourcemanager:azure-resourcemanager-devtestlabs;1.0.0;1.1.0-beta.1
 com.azure.resourcemanager:azure-resourcemanager-hanaonazure;1.0.0-beta.2;1.0.0-beta.3
 com.azure.resourcemanager:azure-resourcemanager-healthcareapis;1.1.0;1.2.0-beta.1
+com.azure.resourcemanager:azure-health-deidentification;1.0.0;1.0.0-beta.1
```

Comment:
Shouldn't the group id just be `com.azure`?

Reply:
yes you are right. I modified it